sdk-tests/pom.xml

```diff
@@ -65,6 +65,12 @@
       <version>2.2.2.RELEASE</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.bettercloud</groupId>
+      <artifactId>vault-java-driver</artifactId>
```

Comment:
This is need for us to programmatically add secrets in integration tests.

Reply:
I think this one should be added as an actual comment inside the xml.

Reply:
ok.

Reply:
Done.